include/openthread/platform/radio.h

```diff
@@ -190,11 +189,12 @@ typedef struct otRadioFrame
          */
         struct
         {
+            const uint8_t *mAesKey;            ///< The key used for AES-CCM frame security.
+            otRadioIeInfo *mIeInfo;            ///< The pointer to the Header IE(s) related information.
```

Comment:
If we move the mIeInfo into mTxInfo only, how could the Rx frame use it?

Reply:
Rx frame will directly read the `TimeIe` in the frame. I have updated the code in the upper layers.